### DISCUSSION TITLE
Filecoin Grant Online Hackathon kicks off!

### DISCUSSION THREAD

**Discussion by Isa-chen** [filecoin-project/community#145] [News and Events]
🇬🇧 > Hi folks! Filecoin Grant Online Hackathon kicks off! You can register now for $50,000 total prize pool! Developers from all over the world can submit IPFS / Filecoin based projects on HackerLink starting from May 1st 2021.

Details in: https://hidorahacks.medium.com/filecoin-grant-online-hackathon-kicks-off-register-now-for-500-000-total-prize-pool-c940a5771a3

🇨🇳 > Hi 大家好！Filecoin Grant线上黑客松开启招募！2021年5月1日起，Filecoin Grant线上Hackathon在DoraHacks开发者平台Hackerlink正式启动！本次Hackathon由Filecoin赞助5万美元奖金池，鼓励开发者们为Filecoin生态贡献idea和早期项目。

详情请见：https://mp.weixin.qq.com/s/YrnJktb0SEZjnPRqBQ2_EQ 欢迎大家踊跃参与!

**Comment by Isa-chen** [comment]
**What you will receive from Filecoin Grant**

- Generous prize: $25K initial prize pool + $25K judges’ prize pool, total $50K!
- Judging panel: judges will observe the whole process, discover potential projects, and vote to allocate the $25K judging bonus pool.
- Filecoin Team Support: Excellent projects will have the opportunity to communicate directly with Filecoin tech team and receive full assistance from them.
- Techn Mentoring Workshop: Technical mentors from Filecoin will explain how to get started with the Filecoin ecosystem through a series of workshops.
- Team/project publicity exposure: Each participating project will have the opportunity to participate in Demo Day live to showcase their projects and gain a lot of exposure.
- Potential investment opportunities: teams have the chance to participate in the project Demo Day live, so that more investors will have a look on your project!

**Schedule**

- Hackathon pre-registration: April 25
- Project Submission: May 1 — July 9
- Project voting: May 1 — July 18 12:00 UTC+0
- Workshop starts: mid-May
- First round of project demos: June 1
- Second round of project demos: June 15
- Third round of demos: July 1
- The final round of focused quick demos: July 17
- End of event: July 18 (UTC+0)

**How to apply?**

1. Log in to hackerlink.io and click on Grant — Filecoin Grant to apply (new users please register first).
2. Join the Filecoin Grant telegram group chat to get latest information about the event. Developers without projects can also team up in the group and turn ideas into projects together! Link: https://t.me/filecoingrant.
3. In order to prevent fake projects from participating in the prize pool, please contact hackerlink(Telegram: @hackerlinkofficial) to verify your project.
4. During the Hackathon, we will conduct multiple rounds of project demos, so the earlier the project is submitted, the more attention it will receive. This will surely increase exposure and attract more votes and funding opportunities.
5. At the end of the Grant Round, before announcing the results, a 24-hour Grace Period will be held to prevent a Sybil-attack. During the Grace Period, each project will be required to contact HackerLink to confirm the authenticity of the project, and after the Grace Period, the project will be able to take out the community donations and the prize pool to match the prize.

**Reply by chief10** [reply]
Hey Isa, 

Who could I reach out to about a potential partnership opportunity? I work with a project called [Meter](www.meter.io) that would allow FIL to traverse other blockchains. We don't charge any fees.

**Reply by realChainLife** [reply]
Hi @chief10 we have a [devgrants](https://github.com/filecoin-project/devgrants) program that would be a great fit for your project. You can also email sonia@fil.org and we explore this further.
